python/cudf/dataframe/categorical.py

```diff
@@ -233,7 +233,7 @@ def _decode(self, value):
     def default_na_value(self):
         return -1
 
-    def find_and_replace(self, to_replace, value):
+    def find_and_replace(self, to_replace, value, all_nan):
```

Comment:
Is `all_nan` ever used?

Reply:
Here, this is never used but in numerical column, `all_nan` is required and we have only one `series.replace()`, which will call the parent class's `replace()` method according to col type(i.e numerical or categorical).